src/main/resources/application.properties

```diff
@@ -0,0 +1,38 @@
+############################################
+###       CLOUD ACCESS                   ###
+############################################
+
+server.address=localhost
+server.port=8887
+
+# Service Registry Core System web-server parameters
+sr_address=127.0.0.1
+sr_port=8443
+
+# CONFIGURE SECURE MODE
+
+# Set this to false to disable https mode
+server.ssl.enabled=true
+
+server.ssl.key-store-type=PKCS12
+server.ssl.key-store=classpath:certificates/sysop.p12
+server.ssl.key-store-password=123456
+server.ssl.key-alias=sysop
+server.ssl.key-password=123456
+server.ssl.client-auth=need
+server.ssl.trust-store-type=PKCS12
+server.ssl.trust-store=classpath:certificates/truststore.p12
+server.ssl.trust-store-password=123456
+
+############################################
+###           DO NOT CHANGE              ###
+############################################
+
+application_system_name=SYSOP
+spring.main.web-application-type=none
+
+############################################
+###           DEFAULT VALUES             ###
+############################################
+
+default_interface_name_list= {'HTTP-SECURE-JSON','HTTP-INSECURE-JSON'}
```

Comment:
if this is a simple comma separated list, you don't need the difficult SpEL expression in the main class.